python_ta/contracts/__init__.py

```diff
@@ -143,29 +144,28 @@ def _check_function_contracts(wrapped, instance, args, kwargs):
     return r
 
 
-def _instance_method_wrapper(wrapped, rep_invariants=None):
-    if rep_invariants is None:
-        return check_contracts
```

Comment:
It's good that you flagged this, although the fix should be a bit different. It's okay if there are no representation invariants, we should still execute the `wrapper` so that the type annotations are checked. So you should revert the change on line 102/104, _keep_ the removal of these two lines, and see my comment below.